components/_patterns/01-atoms/06-buttons/01-button.twig

```diff
@@ -20,7 +20,9 @@
   {% for attribute,value in button_attributes %}
     {{ attribute }}="{{ value }}"
   {% endfor %}
-  href="{{ button_url }}"
+  {% if buttom_url %}
```

Comment:
Functional review revealed this typo—`buttom_url` should have an N at the end, `button_url`. Fix this and I'll get this merged. Thanks!

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/0fc7911b-17b4-44e7-b719-7a8ed87e449c-1a191652-e4fe-4ea3-93fc-3ff6ac9e76a1-v1.png?transparent=1&palette=1&width=246)